lib/driver.js

```diff
@@ -240,13 +240,20 @@ class WebDriverAgentDriver extends BaseDriver {
            !_.includes(commandsToNotProxyToSafari, command);
   }
 
-  async executeCommand (cmd) {
+  async executeCommand (cmd, ...args) {
     if (this.shouldProxyToSafari(cmd)) {
       let safariDriver = await this.getSafariDriver();
-      return await safariDriver.executeCommand(...arguments);
+      return await safariDriver.executeCommand(...args);
```

Comment:
@imurchie previous implementation didnt pass in cmd. but to be consistent this should pass in cmd. Ill update.